docs(kommander): Add distributed authnz pages (#2795)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/latest/operations/accessing-the-cluster/index.md

You will then see Konvoy's operations portal, which offers an overview of cluster status, and shortcuts to several dashboards to addon services such as Grafana.

> These static credentials should be used only to access **operations portal** for configuring an [external identity provider](../../security/external-idps/). Since these are static credentials and there is currently no way of updating them they should be treated only as backup credentials and not used for normal access. Always login with own identity from external identity provider that usually provide additional security features like Multi Factor Authentication.

Reviewer: ```suggestion
Static credentials should only be used to access the **operations portal** for configuring an [external identity provider](../../security/external-idps/). There is currently no way of updating static credentials so they should be used as backup credentials and not used for normal access. Always login with your own identity from an external identity provider. This provides additional security features like Multi Factor Authentication.
```
